### PR TITLE
manual filament cutter first revision

### PR DESCRIPTION
**Pull request by ImChrono**
I’ve added a basic guide on how to set up the filament cutter. The page is still in its early stages, and I haven’t figured out how to add it to the navbar yet. If you could add it, that would be great!

### CODE REVIEW

File: docs/guides/manualcutter.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+**Note**:  
+- If X or Y is set to `-1`, it indicates that the toolhead moves along one axis to reach the parking position.
+The printer will adjust movement accordingly. If both axes are used, define both positions.
+
+### Modifying Settings
+To change values temporarily, use the `SET_CUTTER_SETTINGS` macro. this can be usefull during troubleshooting and testing
+
+```gcode
+SET_CUTTER_SETTINGS PARKING_X=290 PUSHING_SPEED=1700
```
Comment (3DCoded) [inline docs/guides/manualcutter.md:61]: There's a missing ending ``` here.

**Review by 3DCoded — CHANGES_REQUESTED** [review]
The documentation looks good! I found a missing ending ``` at the end of the page.
To add it to the navigation, in `mkdocs.yml`, you can add
```
- guides/manualcutter.md
```
to the `Guides:` section in the `nav:` section.

File: docs/guides/manualcutter.md (hunk 1 above)
Comment (ImChrono) [reply docs/guides/manualcutter.md:61]: completely missed that one 👍

**Comment by 3DCoded** [comment]
Thank you for your contribution to the 3MS project!

**Pull request merged**